tests/recipe_tests/test_XarrayZarrRecipe.py

```diff
@@ -120,7 +131,7 @@ def test_recipe(recipe_fixture, execute_recipe):
     rec = RecipeClass(file_pattern, **kwargs)
     execute_recipe(rec)
     ds_actual = xr.open_zarr(target.get_mapper()).load()
-    xr.testing.assert_identical(ds_actual, ds_expected)
+    xr.testing.assert_identical(drop_execution_context_attrs(ds_actual), ds_expected)
```

Comment:
I'm not a fan of having to include this statement in every test.

Instead, let's write a wrapper around `xr.testing.assert_identical` like this

```python
def assert_identical(ds1, ds2):
    xr.testing.assert_identical(
        drop_execution_context_attrs(ds1),
        drop_execution_context_attrs(ds2)
    )
```

and call it from the individual tests.

Reply:
👍 b781e30